### DISCUSSION TITLE
第4节课作业(24班)

### DISCUSSION THREAD

**Discussion by zhjunqin** [InternLM/Tutorial#382] [作业]
第4次课作业(24班)

**Comment by yansuoyuli** [comment]
https://blog.csdn.net/m0_58431543/article/details/135663067

**Comment by yezhou05** [comment]
https://app.yinxiang.com/fx/8ee78ff1-4e83-4b69-8cfb-aa16b29422b5

**Comment by waurblove** [comment]
https://github.com/waurblove/LLM/blob/main/day4.md

**Comment by kelenlv** [comment]
https://github.com/kelenlv/LLM_notes/blob/main/Day4_0.md

**Comment by flh1107** [comment]
https://www.bilibili.com/read/cv29922453/

**Comment by lindsey-chang** [comment]
https://blog.csdn.net/qq_37397652/article/details/135774896?spm=1001.2014.3001.5502

**Comment by Xinyi-Cheng** [comment]
https://zhuanlan.zhihu.com/p/679371266

**Comment by ch521-od** [comment]
https://blog.csdn.net/qq_74261455/article/details/135752724?spm=1001.2014.3001.5502

**Comment by robotzm** [comment]
https://blog.csdn.net/weixin_48319544/article/details/135796433?spm=1001.2014.3001.5501
